### DISCUSSION TITLE
Renaming repositories

### DISCUSSION THREAD

**Discussion by caiopiccirillo** [Open-Source-Digital-Twin/digital-twin-playground#63] [General]
I've being thinking about the names of our repositories and I'd like to rename them as follows:
1. From [motion-control-playground](https://github.com/Open-Source-Digital-Twin/motion-control-playground) to `digital-twin-playground`.
2. From [inverted-pendulum-controller](https://github.com/Open-Source-Digital-Twin/inverted-pendulum-controller) to `furuta-pendulum-controller`.

Rationale:
1. The project is not only aimed for motion control, but for interacting with digital twins.
2. The proper term for the rotational inverted pendulum is "Furuta pendulum". So it looks shorter and nicer.

@LLagu I'd like your opinion on this.

**Comment by LLagu** [comment]
You're making very good points.

I'm all for `motion-control-playground` to `digital-twin-playground` name change.
I like the `furuta-pendulum-controller` as well but do you think we should mention that it's a digital twin furuta in the name? I saw that there are other rotary inverted pendulum/furuta repos but they seem to be mostly hardware related. Or maybe it's sufficient to make the distinction in the README.

**Reply by caiopiccirillo** [reply]
That's a good point, we can make this clear on the readme, otherwise the name would be huge 😋

**Reply by LLagu** [reply]
Perfect! :)
